### ISSUE TITLE
Adds tabs automatically, in places where they are not needed

### ISSUE THREAD

**Issue by t-zander** [Microsoft/vscode#55863]
In html I have markup like this:
```
<div id="outer">
  <div  id="inner">
   Some content here
  </div>
</div>
```
For some strange reason, vscode indents div with id inner with TWO tabs instead of one. When I delete one tab and save, it adds second tab autamatically. That's really annoying and breaks my structure. Any ways to solve that?

**Comment by mathdeziel** [comment]
If I understand correctly @t-zander, you want you code to look like this :

``````

**Comment by ramya-rao-a** [comment]
@mathdeziel is right.
@t-zander let us know if the above doesnt help

**Comment by t-zander** [comment]
Sorry for the late reply.
I mean the following.
I created html
It look like this
![image](https://user-images.githubusercontent.com/34350867/44161399-df4a7380-a0c5-11e8-812c-b05672bc66b7.png)
But I want it to be like this(tab spaces):
![image](https://user-images.githubusercontent.com/34350867/44161438-01dc8c80-a0c6-11e8-88c3-075d15c6d1ca.png)
I have tab=2 spaces but for some reason it make 2 tabs when I clisk save or when it autosaves this.

**Comment by t-zander** [comment]
It was already set, still have this problem

**Comment by ramya-rao-a** [comment]
In the first 2 screenshots, the status bar shows "Tab Size: 2" which means it was using tabs to indent

**Comment by t-zander** [comment]
Ok, I did that.
I still had that trouble...
I am bored with it.
I tried 2 spaces, or tabs, I tried anything.
Still, when I try to do it like this
![image](https://user-images.githubusercontent.com/34350867/44959908-24143e00-aeff-11e8-8ab6-c40afbcccc0b.png)
When I click ctrl+s it becomes this...
![image](https://user-images.githubusercontent.com/34350867/44959915-30989680-aeff-11e8-90bc-e9d7d148415b.png)
By the way for some reason it interprets html as django template and so I can't even comment something out cause instead of normal comments I get this...
![image](https://user-images.githubusercontent.com/34350867/44959931-5de54480-aeff-11e8-9a74-88d99b094b36.png)
